Assets/LeapMotionModules/UI/Interaction/InteractionBehaviourBase.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Leap;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public abstract class InteractionBehaviourBase : MonoBehaviour {
+
+    public InteractionManager interactionManager;
+
+    protected Rigidbody _body;
+    public Rigidbody Rigidbody { get { return _body; } }
```

Comment:
would be nice to name this rigidbody instead, to follow property naming conventions.